app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: float,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        total_cache = 0.0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_cache += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return round(total_cache, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class * (
+                self.clean_power - car.clean_mark
+            ) * (
+                self.average_rating / self.distance_from_city_center
+            ), 1
```

Comment:
Format your calculation expressions according to examples from the checklist avoiding line breaking with `()` and preferring line breaking by math operators